spec/icd/api/authorize_token_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Icd
+  module Api
+    RSpec.describe AuthorizeToken do
+      before :suite do
+        described_class.configure do |conf|
+          conf.client_id = 'client_id'
+          conf.client_secret = 'client_secret'
+        end
+      end
+      describe '#retrieve_access_token' do
+        subject { described_class.call }
+
+        it 'sends request to token endpoint' do
```

Comment:
maybe add request matchers?

Reply:
But they are in the cassette:
spec/vcr/icd/api/authorize_token.yml
So the vcr_cassette contains the right URL and also the response payload.